lib/domain/model/extensions/contact/presentation_contact_list_extension.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:fluffychat/domain/model/extensions/contact/presentation_contact_extension.dart';
+import 'package:fluffychat/presentation/model/presentation_contact.dart';
+import 'package:fluffychat/presentation/model/presentation_search.dart';
+
+extension PresentaionContactListExtension on List<PresentationContact> {
```

Comment:
not necessary

Reply:
- Why not necessary? 
It's converted `List<PresentationContact>` to `List<PresentationSearch>`